docs/mounting.md

```diff
@@ -22,6 +22,10 @@ Console:
 
     GOOGLE_APPLICATION_CREDENTIALS=/path/to/key.json gcsfuse [...]
 
+On OS X and Linux you can also use fstab option in `/etc/fstab` system file:
```

Comment:
Ah you're right. In that case, could you actually move this information to the fstab section of this file? So it says something like

> If you need to specify custom credentials in your fstab, use the `key_file=/path/to/file` option.